bitnami/discourse/2/debian-11/rootfs/opt/bitnami/scripts/libdiscourse.sh

```diff
@@ -249,6 +249,7 @@ discourse_create_conf_file() {
     discourse_conf_set "redis_host" "$DISCOURSE_REDIS_HOST"
     discourse_conf_set "redis_port" "$DISCOURSE_REDIS_PORT_NUMBER"
     discourse_conf_set "redis_password" "$DISCOURSE_REDIS_PASSWORD"
+    discourse_conf_set "redis_use_ssl" "$DISCOURSE_REDIS_USE_SSL"
```

Comment:
Related to the comment above, could you adapt this line in the following way?

```
is_boolean_yes "$DISCOURSE_REDIS_USE_SSL" && discourse_conf_set "redis_use_ssl" true
```

Since the default value is `false`, it should continue working fine.

Reply:
Thx. Changes applied...